src/components/PrivacyDisclaimer.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Button } from 'antd';
+
+const Footer = styled.div`
+  width: 100vw;
+  overflow: hidden;
+  padding: 12px 40px;
+  display: flex;
+  border-top: 1px solid #efefef;
+  background-color: #fff;
+  align-items: center;
+  p {
+    flex: 1;
+    margin-right: 12px;
+    margin-bottom: 0px;
+  }
+  button.dismiss {
+    margin-right: 0px;
+  }
+  button {
+    margin-right: 12px;
+  }
+`;
+
+export const PRIVACY_ACCEPT = 'acceptPrivacy';
+
+class PrivacyDisclaimer extends React.Component {
+  componentDidMount() {
+    if (localStorage.getItem(PRIVACY_ACCEPT) === null) {
+      setTimeout(() => {
+        localStorage.setItem(PRIVACY_ACCEPT, 'true');
+        this.forceUpdate();
+      }, 30000);
+    } else if (navigator.doNotTrack === '1') {
+      // eslint-disable-next-line no-console
+      console.warn(
```

Comment:
So I was not sure if I should have a popup to ask the user to disable do not track since we would really, really like to track the user's workflow for better analytics. But having a popup would be really intrusive, theres no policy on this yet, so its def up for debate.

Reply:
You can use this component to show the DNT message, possibly without the option to dismiss the the message if DNT is enabled. Or, instead of removing the dismissal feature, just persist the dismissal into the app state or session storage so the message will be recurring)?

Reply:
We can, it's just I don't see any other website doing this since I didn't know I had DNT the whole time and no website warned me :/ also since it's a browser wide setting should a user have to turn it off just for discovery?